File: src/main/java/com/wxl/crawlerdytt/processor/DyttProcessor.java
package com.wxl.crawlerdytt.processor;


import us.codecraft.webmagic.Page;

/**
 * Create by wuxingle on 2020/5/10
 * 页面匹配处理
 */
public interface DyttProcessor {

    void process(Page page);

    boolean match(Page page);
}

File: src/main/kotlin/com/wxl/dyttcrawler/processor/DyttProcessor.kt
package com.wxl.dyttcrawler.processor

import us.codecraft.webmagic.Page

/**
 * Create by wuxingle on 2021/10/12
 * 页面匹配处理
 */
interface DyttProcessor {

    fun process(page: Page)

    fun match(page: Page): Boolean
}